Samples/Gtk3/GtkMinimal/DemoWindow.cs

```diff
@@ -14,7 +14,7 @@ public class DemoWindow : ApplicationWindow
         private Notebook notebook;
 
         public DemoWindow(Application application) : this(application, new Builder("demo_window.glade")) { }
-        private DemoWindow(Application application, Builder builder) : base(builder.GetObject("root"))
+        private DemoWindow(Application application, Builder builder) : base(builder.GetObject("root"), false)
```

Comment:
This sample should be updated to use the source generator `[Template(...)]` attribute, but that's for another PR.

Reply:
Yes this is for another PR. But we need to decide if we want to have a sample using the `Builder` API. This API does currently not support the `TemplateAttribute` as it is not using composite widgets.